packages/relay/src/lib/services/metricService/metricService.ts

```diff
@@ -102,19 +138,23 @@ export default class MetricService {
    */
   constructor(
     logger: Logger,
-    sdkClient: SDKClient,
+    hapiService: HAPIService,
     mirrorNodeClient: MirrorNodeClient,
     hbarLimiter: HbarLimit,
     register: Registry,
     eventEmitter: EventEmitter,
   ) {
     this.logger = logger;
-    this.sdkClient = sdkClient;
+    this.hapiService = hapiService;
```

Comment:
As we remove the class property this line should be removed we only need the hapiService instance to get the sdk client in the line below